boards/arm/frdm_k64f/frdm_k64f.dts

```diff
@@ -202,6 +202,13 @@ zephyr_udc0: &usbotg {
 	num-bidir-endpoints = <8>;
 };
 
+/* zephyr_uhc0: &uhc1 {
```

Comment:
Why is this commented out?

Reply:
I see the auto-test uses the frdm_k64f board + max3421e. If I enable the zephyr_uhc0 in frdm_k64f device tree default, the auto-test case fail because there are two zephyr_uhc0.
So I enable the frdm_k22f too in this pr.

Reply:
Ah- I see. Maybe we need to make the auto-test case for the frdm_k64f board + max3421e into one run via an overlay file. See here for an example of a testcase like this in Zephyr: https://github.com/zephyrproject-rtos/zephyr/blob/9f7695dda0c3068ab6738a9f575a1c6dd363126d/tests/drivers/w1/w1_api/testcase.yaml#L24

Reply:
I don't know where to find the USB test related config file. Could we resolve it by future PR not this one? Or I can delete the frdm_k64f support in this PR?

Reply:
We can resolve it in a future PR, and remove the K64F support from this PR. If you'd like to create an issue highlighting this and assign it to me, feel free to do so

Reply:
The new issue is here https://github.com/zephyrproject-rtos/zephyr/issues/70013, I cannot assign it to you, I cannot change 'Assignees'.